solr/core/src/test/org/apache/solr/response/transform/TestChildDocTransformer.java

```diff
@@ -66,34 +66,34 @@ public void testAllParams() throws Exception {
   private void testChildDoctransformerXML() {
     String test1[] = new String[] {
         "//*[@numFound='1']",
-        "/response/result/doc[1]/doc[1]/str[@name='id']='2'" ,
-        "/response/result/doc[1]/doc[2]/str[@name='id']='3'" ,
-        "/response/result/doc[1]/doc[3]/str[@name='id']='4'" ,
-        "/response/result/doc[1]/doc[4]/str[@name='id']='5'" ,
-        "/response/result/doc[1]/doc[5]/str[@name='id']='6'" ,
-        "/response/result/doc[1]/doc[6]/str[@name='id']='7'"};
+        "/response/result/doc[1]/arr[@name='_childDocuments_']/doc[1]/str[@name='id']='2'" ,
```

Comment:
One major difference is the way nested XML is returned, since now it has a key it belongs to.
Thought it would make the review process easier for you if I pointed these differences out.